code-scanning/checkmarx-one.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# This is a basic workflow to help you get started with Using Checkmarx One Action
```

Comment:
Can we have more details what this template/action does, how to make use of it, prerequisites and docs link?

Reply:
Sure, should put them within the comments as well ?

Reply:
What does checkmarx one action do? Can we some some description around that?